docs/src/analysis/sensitivity.md

```diff
@@ -378,26 +378,27 @@ will plot the expanded system).
 Adjoint sensitivity analysis is used to find the gradient of the solution
 with respect to some functional of the solution. In many cases this is used
 in an optimization problem to return the gradient with respect to some cost
-function. It is equivalent to "backpropogation" or reverse-mode automatic
+function. It is equivalent to "backpropagation" or reverse-mode automatic
 differentiation of a differential equation.
 
 ### Adjoint Sensitivity Analysis via adjoint_sensitivities
 
-This adjoint requires the definition of some scalar functional ``g(u,p,t)``
-where ``u`` is the (numerical) solution to the differential equation.
+This adjoint requires the definition of some scalar functional ``g(u,p)``
+where ``u(t,p)`` is the (numerical) solution to the differential equation
```

Comment:
p,t is used everywhere in code, so keep the ordering consistent.

Reply:
I am sorry, yes, please use an ordering with is consistent with the code, but use it consistently.